components/ResidentPage/AllocationHeader.tsx

```diff
@@ -0,0 +1,45 @@
+import { Allocation } from 'types';
+import Link from 'next/link';
+
+interface Props {
+  allocations: Allocation[];
+}
+
+const AllocationHeader = ({ allocations }: Props): React.ReactElement => {
+  const linkTeam = allocations[0].allocatedWorkerTeam && (
+    <Link href={`/teams/${allocations[0].allocatedWorkerTeamId}`}>
+      <a
+        data-testid="teamLink"
+        className={`lbh-link lbh-link--no-visited-state`}
+      >
+        {allocations[0].allocatedWorker
+          ? `(${allocations[0].allocatedWorkerTeam})`
+          : allocations[0].allocatedWorkerTeam}
+      </a>
+    </Link>
+  );
+
+  const moreLink = allocations?.length > 1 && (
+    <Link href={`/residents/${allocations[0].personId}/allocations`}>
+      <a
+        data-testid="allocationLink"
+        className={`lbh-link lbh-link--no-visited-state`}
+      >
+        {allocations?.length === 2
```

Comment:
Should this be `allocations?.length =< 2`?

Reply:
It should be correct, as if the allocations are 2 we display the first one with long description + "one other"

If it's more than 2 (the "else") it says the length -1 (so, not counting the one with long description)